frontend/src/routes/manage/Realm/Content/Edit/EditMode/util.tsx

```diff
@@ -1,8 +1,30 @@
 import React from "react";
+import { useTranslation } from "react-i18next";
+import { useFormContext } from "react-hook-form";
+
 
 export const Heading: React.FC = ({ children }) => <h3 css={{
     marginTop: 8,
     marginBottom: 4,
 }}>
     {children}
 </h3>;
+
+type ShowTitleProps = {
+    showTitle: boolean;
+};
+
+export const ShowTitle: React.FC<ShowTitleProps> = ({ showTitle }) => {
+    const { t } = useTranslation();
+    const { register } = useFormContext<ShowTitleProps>();
+
+    console.log(showTitle);
+
```

Comment:
ooopilie poopelie

Reply:
Maybe we want to enable the "console" lint. I just checked, we only have very few legit cases of `console` in our code base that we could easily annotate with `// eslint is ok, dont worry about it`.

Reply:
I will add it.